juju/juju
# Create pull secrets for sidecar containers.

Adds image pull secrets for sidecar charms with private images such as from charmhub.

## QA steps

Deploy charm with private repo images.
Check image pull secrets exist and images pull.
Refresh charm with public repo images.
Check image pull secrets are cleaned up.

## Documentation changes

N/A

## Bug reference

https://bugs.launchpad.net/juju/+bug/1934416
